project3/index.html

```diff
@@ -0,0 +1,251 @@
+                            fill="#fff" />
+                        <path
+                            d="M18 103a7 7 0 100 14 7 7 0 000-14zm0 10c-1.654 0-3-1.346-3-3s1.346-3 3-3 3 1.346 3 3-1.346 3-3 3zM3.227 14.283l-.5 3.073A4 4 0 008.608 21.5L11 20.18l2.391 1.321a3.995 3.995 0 004.239-.23 4 4 0 001.644-3.914l-.501-3.073 2.09-2.142a3.999 3.999 0 00-2.259-6.747l-2.727-.417L14.621 2.3a4 4 0 00-7.242 0L6.123 4.977l-2.727.417a4 4 0 00-2.259 6.747l2.09 2.142zm5.61-5.675L11 4l2.163 4.608 4.837.74-3.5 3.587.826 5.065L11 15.608 6.674 18l.826-5.065L4 9.348l4.837-.74zM124.605 103.978l-6.874-1.052-3.11-6.626a4 4 0 00-7.242 0l-3.11 6.626-6.874 1.052a4 4 0 00-2.258 6.747l5.09 5.217-1.209 7.414a4 4 0 005.883 4.145l6.099-3.373 6.099 3.373a3.999 3.999 0 005.883-4.145l-1.21-7.414 5.09-5.217a4 4 0 00-2.257-6.747zm-7.106 10.615l1.535 9.407-8.034-4.442-8.034 4.442 1.534-9.407-6.5-6.661 8.983-1.374L111 98l4.017 8.558 8.983 1.374-6.5 6.661zM113.88 4.23c.14 0 .28-.02.42-.05a2 2 0 10-2.38-1.53c.2.94 1.03 1.58 1.96 1.58zM122.33 6.26c.58 0 1.15-.25 1.55-.73.7-.86.57-2.12-.28-2.82a2.01 2.01 0 00-2.82.28c-.7.86-.57 2.12.28 2.82.38.3.82.45 1.27.45zM113.28 23.69c.15.04.31.05.46.05.91 0 1.73-.62 1.94-1.55.25-1.07-.42-2.15-1.49-2.4-1.08-.25-2.16.42-2.4 1.5-.25 1.08.42 2.15 1.49 2.4zM122.22 21.83a1.994 1.994 0 001.56-3.24 1.99 1.99 0 00-2.81-.32c-.86.68-1 1.94-.32 2.81.4.49.98.75 1.57.75z"
+                            fill="#fff" />
+                        <circle cx="126" cy="12" r="2" fill="#fff" />
+                        <path
+                            d="M106.96 18.28c.29 0 .58-.06.86-.19 1-.47 1.42-1.67.95-2.66-.48-1-1.67-1.43-2.67-.95-1 .47-1.42 1.66-.95 2.66.35.72 1.06 1.14 1.81 1.14zM106.14 9.38c.29.14.59.21.88.21a2.004 2.004 0 00.89-3.8 2.002 2.002 0 00-1.77 3.59z"
+                            fill="#fff" />
+                        <circle cx="116" cy="12" r="2" fill="#fff" />
+                        <circle cx="10" cy="76" r="2" fill="#fff" />
+                        <circle cx="10" cy="38" r="2" fill="#fff" />
+                        <path d="M12 46a2 2 0 00-4 0v22a2 2 0 004 0V46z" fill="#fff" />
+                        <circle cx="118" cy="76" r="2" fill="#fff" />
+                        <circle cx="118" cy="38" r="2" fill="#fff" />
+                        <path d="M120 46a2 2 0 00-4 0v22a2 2 0 004 0V46z" fill="#fff" /></svg>
+
+                    <label for="intelligience">Intelligience</label>
```

Comment:
Note this typo which breaks the connection between your `label` and `input`:
```suggestion
                    <label for="intelligence">Intelligence</label>
```